picobot/painter.py

```diff
@@ -85,7 +87,36 @@ def draw_message(
 ):
     x0 = xy[0] + MSG_PADDING_H
     y0 = xy[1] + MSG_PADDING_V + user_size[1] + LINE_SPACE
-    txt_draw.text((x0, y0), text, font=font, fill=fill)
+    symbola_font = ImageFont.truetype(font=str(FONT_DIR / 'Symbola.ttf'), size=16)
+    emoji_locations = emoji_lis(text)
+    indexes = [0]
+
+    # some encoding shenanigans for complex emojis
```

Comment:
Not sure if I like such a cryptic comment. Mind linking to something that explains the idea (or briefly explaining it, if possible to explain in 2~3 comment lines)?

Reply:
Why don't you let me be lazy, @JPTIZ ? :disappointed: 
Done.